### ISSUE TITLE
order of syntax commands - 'label' after 'format'

### ISSUE THREAD

**Issue by LCisannoyed** [markdly/conquestr#9]
is it possible to change the output order? so the 'label' command comes _after_ the 'format' command

Currently code produces this order. Which CQ wont read, but has no issues if I swap order of commands.

`reset;`
`title Numeracy_Y3_a1 ;`
`data Year_3_Calibration_Extract_with_DAC_redaction_20180306.txt ; `
**`label << Numeracy_Y3_a1.lab;`**
**`format  responses 201, 202, 203, 204, 205, 206, 207, 208, 209, 210, 211, 212;`**
`codes 0,1,2,3,4,5,6,7,8,9;`
`key 111111111111 ! 1;`
`model item;`
`estimate ! iter=1000, nodes=30;`
`itanal >> Numeracy_Y3_a1.itn;`

**Comment by markdly** [comment]
Yes we can update the default order generated by `cqc_cmds()` so it works as is. I'll work on this now...

**Comment by LCisannoyed** [comment]
its not urgent. 

And it might just be me / my CQ.exe because other times it will accept your ordering. I think this never used to be a problem in older CQ versions
